tracing/src/macros.rs

```diff
@@ -806,6 +806,9 @@ macro_rules! event {
 /// This is similar to [`enabled!`], but queries the current collector specifically for
 /// an event, whereas [`enabled!`] queries for an event _or_ span.
 ///
+/// Although it shares the name, this *does not* call [`Collect::event_enabled`].
```

Comment:
i think this link needs a reference; the CI docs build is failing: https://app.netlify.com/sites/tracing-rs/deploys/62b78349e34bef000848bf14#L305

note that `Collect` isn't imported in this module, so adding
```rust
/// [`Collect::event_enabled`]: crate::Collect::event_enabled
```
at the end of the doc comment should fix this.